actionmailbox/app/controllers/rails/conductor/action_mailbox/inbound_emails_controller.rb

```diff
@@ -20,7 +20,7 @@ def create
 
     private
       def new_mail
-        Mail.new(params.require(:mail).permit(:from, :to, :cc, :bcc, :in_reply_to, :subject, :body).to_h).tap do |mail|
+        Mail.new(params.require(:mail).permit(:from, :to, :cc, :bcc, :in_reply_to, :subject, :body, attachments: []).to_h).tap do |mail|
```

Comment:
Can you add a test for this so we can catch unpermitted params in the future?

Reply:
Hi @eileencodes, thanks for the note. I couldn't find the appropriate tests last night so didn't get a chance to add one in. Oddly, I just found a reference to the attachments here: 

https://github.com/rails/rails/blob/a40da823d729296514f310d13a09ce8e0b37b699/actionmailbox/test/controllers/rails/action_mailbox/inbound_emails_controller_test.rb#L54-L66

Kind of strange it didn't catch the issue though... I'll take a closer look as soon as I can.